src/main/java/com/avioconsulting/mule/connector/akv/provider/api/model/OAuthError.java

```diff
@@ -16,7 +16,7 @@
   private List<Integer> errorCodes;
 
   @SerializedName(value = "timestamp")
-  private Date timestamp;
+  private String timestamp;
```

Comment:
Is there a good reason to change this from a Date to a String?

Reply:
I got a deserialization when manually testing with the Microsoft API. This fixed it.